python/tests/integration_tests/test_client.py

```diff
@@ -177,6 +177,15 @@ def test_datasets(langchain_client: Client) -> None:
         langchain_client.list_examples(dataset_id=new_dataset.id)  # type: ignore
     )
     assert len(examples2) == 2
+    langchain_client.create_example(
+        inputs={},
+        outputs=None,
+        dataset_id=new_dataset.id,
+    )
+    examples3 = list(
```

Comment:
can we check the value of the last created example

Reply:
Done